Tests/LibCore/TestLibCoreMimeType.h

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright (c) 2025, Andreas Kling <andreas@ladybird.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/Vector.h>
+
+void check_filename_mimetype(AK::Vector<StringView>const&, StringView const&);
```

Comment:
A header file for the test is not necessary. Make the helper function file-static instead.